src/zenml/integrations/seldon/seldon_client.py

```diff
@@ -292,8 +274,10 @@ class SeldonDeployment(BaseModel):
         status: Seldon Deployment status.
     """
 
-    kind: str = Field(SELDON_DEPLOYMENT_KIND, const=True)
-    apiVersion: str = Field(SELDON_DEPLOYMENT_API_VERSION, const=True)
+    kind: Literal["SeldonDeployment"] = "SeldonDeployment"
+    apiVersion: Literal["machinelearning.seldon.io/v1"] = (
+        "machinelearning.seldon.io/v1"
+    )
```

Comment:
If this is constant, why do we need this in the model at all? Maybe kick it out to the constants?

Reply:
not really sure why it was implemented like this before. perhaps @safoinme can explain if there is a difference.